File: 프로그래머스/0/120891. 369게임/369게임.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int order) {
    int answer = 0;
    string ord = to_string(order);
    for(auto o : ord)
    {
        if((o - '0') != 0 && (o - '0') % 3 == 0)
            answer++;
    }
    return answer;
}

File: 프로그래머스/0/120891. 369게임/369게임.java
class Solution {
    public int solution(int order) {
        int answer = 0;
        String ord = Integer.toString(order);
        
        for(int i = 0; i < ord.length(); i++) {
            if((ord.charAt(i) - '0') != 0 && (ord.charAt(i) - '0') % 3 == 0)
                answer++;
        }
        
        return answer;
    }
}